experimental/common-expression-language/README.md

```diff
@@ -1,33 +1,41 @@
+|<	        | Less than. A < B	                                                    |[less][10]                         |
+|>= 	    | Greater than or equal. A >= B	                                    |[greaterOrEquals][11]              |
+|<=	        | Less than or equal. A <= B	                                        |[lessOrEquals][12]                 |
+
+### Other operators and expression syntax
 
 | Operator	|                                  Functionality                                            |   Prebuilt function equivalent    |
 |-----------|-------------------------------------------------------------------------------------------|-----------------------------------|
-|+          |Arithmetic operator – addition. E.g. A + B	                                                |[add][1]                           |
-|-	        |Arithmetic operator – subtraction. E.g. A – B	                                            |[sub][2]                           |
-|unary +    |Arithmetic operator – positive E.g. +1, +A	                                                |N/A                                |
-|unary -	|Arithmetic operator – negative value E.g. –2, -B	                                        |N/A                                |
-|*	        |Arithmetic operator – multiplication. E.g. A * B	                                        |[mul][3]                           |
-|/	        |Arithmetic operator – division. E.g. A / B	                                                |[div][4]                           |
-|^	        |Arithmetic operator – exponentiation. E.g. A ^ B	                                        |[exp][5]                           |
-|%	        |Arithmetic operator – modulus. E.g. A % B	                                                |[mod][6]                           |
-|==	        |Comparison operator – equals. E.g. A == B	                                                |[equals][7]                        |
-|!=	        |Comparison operator – Not equals. E.g. A != B	                                            |[not][8]([equals][7]())            |
-|>	        |Comparison operator – Greater than. A > B	                                                |[greater][9]                       |
-|<	        |Comparison operator – Less than. A < B	                                                    |[less][10]                         |
-|>= 	    |Comparison operator – Greater than or equal. A >= B	                                    |[greaterOrEquals][11]              |
-|<=	        |Comparison operator – Less than or equal. A <= B	                                        |[lessOrEquals][12]                 |
 |&	        |Concatenation operator. Operands will always be cast to string – E.g. A & B	            |N/A                                |
 |&&	        |Logical operator – AND. E.g. exp1 && exp2	                                                |[and][13]                          |
```

Comment:
May be we can group the logical operators into their own section?

We also support ! for logical NOT operation.